x-pack/plugins/search_playground/server/utils/exceptions.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface SearchPlaygroundOpenAIResponseError {
+  status: number;
+  code: string;
+}
+
+export const isInvalidApiKeyException = (error: SearchPlaygroundOpenAIResponseError) =>
```

Comment:
Here is more error code provided [Error codes](https://platform.openai.com/docs/guides/error-codes/error-codes)
Like "You exceeded your current quota, please check your plan and billing details". 
I guess we should review error codes and cover relevant cases for us

Reply:
Or... I found that openAI connector has their own error messages, as we planning to use it we probably can use its method to define the type of error and return the text of it
https://github.com/elastic/kibana/blob/fed4af1f31d2817dc685ff5469160764f877b6b2/x-pack/plugins/stack_connectors/server/connector_types/openai/openai.ts#L95